meetup-organizing.md

```diff
@@ -0,0 +1,38 @@
+# Organizing a meetup
+
+The following details the basic process for working with the Quorum to plan a meetup, confirm a speaker and topic, and provide any needed information or materials for the meetup event.
+
+The suggested timeframe for completing each of the preparation steps are:
+
+|   Step    |   Timeframe    |
+|  ---  |  ---  |  ---  |  ---  |
+|   Confirm details    |   1 month before    |
+|   Create event    |   3 weeks before    |
+|   Publicize event    |  2 weeks before     |
+
+## Confirm details
+
```

Comment:
**Suggestion (non-blocking):** Before the confirm details section, it might be good to have a section for planning the meeting. Here's some copy that could go into that section (feel free to revise or reject):

To plan the meetup, begin by brainstorming possible speakers or presenters that you'd like to consider for your meetup. As you are brainstorming, ask yourself a few questions:

- **Is there anyone in my immediate social network that would make a great speaker, such as a friend, co-worker, or a member of my local meetup?** You'd be surprised who you might know! Keep in mind that speakers don't have to be technical writers by profession: developers, UX designers, product managers, Scrum masters, or other related professionals all have useful insights to share.
- **Have I read an interesting book on a documentation-related topic that I'd like to hear more about?** Speakers are usually willing to promote and discuss their books, especially to receptive audiences. Most people you approach are likely to say yes, so go ahead and dream big!
- **Have I heard or watched any interesting presentations or podcast episodes outside of Write the Docs that I'd like to hear more about? Who seems to be on the speaking circuit right now?** Sometimes you can find good speakers that have presented at conferences or other local meetups, such as the Society of Technical Communicators, Confab, or other technology- or content-oriented conferences. Speakers who have already given or prepared a presentation are usually more than happy to give the same presentation again.
- **Did someone in the WTD community solve a problem in an interesting way or develop a novel approach to a documentation task that you want to learn more about?** Sometimes people in the Write the Docs community have interesting approaches to solving documentation problem that you might want to hear more about. Feel free to reach out to members of our community that seem like they might be interested in providing more details.
- **Does someone who works for a company that makes documentation tools want to speak to our audience?** While we don't want every presentation to turn into a product demo, members of our community might be genuinely interested in hearing about new products or tools that are being developed to help solve documentation problems. Some product demos can be quite interesting and these kind of speakers (of course) will say yes to the opportunity to speak.

If you're stuck, Alyssa will be more than happy to work with you to brainstorm possible speakers and has some decent connections to some good speakers personally. Feel free to reach out to her!

Also, in the future, we hope to provide a speaker database to help assist with this part of the process. If you'd like to volunteer to help organize this database, let us know!

Reply:
I totally blanked on this section. Thanks.